### ISSUE TITLE
go generator doesn't handle JS providers

### ISSUE THREAD

**Issue by stillmatic** [steebchen/prisma-client-go#877]
I have a prisma file setup in a Go + TS monorepo as 

```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["jsonProtocol", "views"]
}

generator golang {
  provider        = "go run github.com/steebchen/prisma-client-go"
  previewFeatures = ["views"]
}
```

Running `npx prisma migrate dev --schema packages/db/prisma/schema.prisma` works

```
✔ Generated Prisma Client (4.14.0 | library) to ./node_modules/@prisma/clie
nt in 120ms
✔ Generated Prisma Client Go to ./packages/db/prisma/db in 1.28s
```

However, running equivalent (just generate) with Go CLI fails

```
Error: Generator "/home/hua/code/monorepo/node_modules/@prisma/client/generator-build/index.js" failed:

node:internal/modules/cjs/loader:936
  throw err;
  ^
Error: Cannot find module '/home/hua/code/monorepo/--max-old-space-size=8096'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function._resolveFilename (pkg/prelude/bootstrap.js:1951:46)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.runMain (pkg/prelude/bootstrap.js:1979:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

panic: could not run [generate --schema packages/db/prisma/schema.prisma]: exit status 1

goroutine 1 [running]:
main.main()
        /home/hua/go/pkg/mod/github.com/steebchen/prisma-client-go@v0.18.0/main.go:29 +0x34b
exit status 2
```

When I comment out the `prisma-client-js` generator, the golang command works fine. This is on a WSL / amd64 instance. 

I am not blocked on this, since the npx command is able to generate correctly. 

Edit -- this may be a dupe of #455

**Comment by steebchen** [comment]
Yeah so basically the Go client is a custom generator which means it's supported, however the JS generator is built into Prisma so the Go client can't directly invoke it. So the best thing to do here is using npx to generate both clients. If you do want to generate the Go client with Go only, you should use a single generator and provide the `provider` variable via an env var depending on which client you want to generate.

**Comment by stillmatic** [comment]
Makes sense! I think having a note in the docs to that effect would be helpful, or perhaps handling it in the code (refusing to run the JS generator), but docs are probably sufficient

**Comment by steebchen** [comment]
yes will be done via #880 and #455, closing this in favor of these

**Comment by steebchen** [comment]
related: https://github.com/prisma/prisma/issues/10564
